content/blog/ferret-v0.11.0.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Ferret v0.11.0"
+subtitle: "More features"
+draft: false
+author: "Vladimir Fetisov"
+authorLink: "https://github.com/3timeslazy"
+date: "2020-05-19"
+---
+
+Hello fellow miners, **[Ferret v0.11.0](https://github.com/MontFerret/ferret/releases/tag/v0.11.0)** has been **finally** released!
+
+In this release, we mostly focused on our full-time jobs and families.
+
+But despite this, We have done a lot of cool features and projects.
+
+Let’s take a look!
+
+--- 
+But there is one word before starting:  
```

Comment:
Remove the ``But there is one word before starting:``.
And move the entire intro to the top. (Let's start the RN with this notion of COVID19)